ocs_ci/ocs/ui/acm_ui.py

```diff
@@ -545,6 +555,53 @@ def create_cluster(self):
             self.deployment_status = "Creating"
             return
 
+    def add_different_pod_network(self):
+        """
+        Edit online cluster yaml to add network info
+
+        """
+
+        def _reset():
+            for device in actions.w3c_actions.devices:
+                device.clear_actions()
+
+        self.driver.execute_script(
+            "return document.querySelector('div.yamlEditorContainer')"
+        )
+        actions = ActionChains(self.driver)
+        yaml_first_line = "apiVersion: v1"
+        for _ in range(0, 3):
+            actions.send_keys(Keys.TAB).perform()
+            time.sleep(1)
+            _reset()
+        for _ in range(len(yaml_first_line)):
+            actions.send_keys(Keys.ARROW_RIGHT).perform()
+            time.sleep(1)
+            log.debug(
```

Comment:
This seems more like a comment than a debug statement

Reply:
done